src/components/Search/DaoSearch/index.tsx

```diff
@@ -38,22 +47,28 @@ const DaoSearch = ({ content }: { content?: string }) => {
         },
       })
     } else {
-      searchNervosDaoTransactions(query, dispatch, (isSuccess: boolean) => {
-        if (isSuccess) {
-          clearSearchInput(inputElement)
-          setShowReset(true)
-        } else {
-          dispatch({
-            type: AppActions.ShowToastMessage,
-            payload: {
-              message: i18n.t('toast.result_not_found'),
-            },
-          })
-        }
-      })
+      searchNervosDaoTransactions(query, dispatch)
     }
   }
 
+  useEffect(() => {
+    console.log(transactionsStatus)
```

Comment:
does this log need to be output in a production environment?

Reply:
Sorry, I will remove it

Reply:
Updated https://github.com/nervosnetwork/ckb-explorer-frontend/pull/580/commits/827bd348bcbdea7d85c491e1fd8f724c851e2bc7